SAPHana/ra/SAPHanaController

```diff
@@ -1305,13 +1305,16 @@ function check_for_primary() {
                return $HANA_STATE_STANDALONE;;
            * )
                super_ocf_log err "ACT: check_for_primary:  we didn't expect node_status to be: <$node_status>"
-               dump=$( echo $node_status | hexdump -C );
-               super_ocf_log err "ACT: check_for_primary:  we didn't expect node_status to be: DUMP <$dump>"
+               if [ -n "$node_status" ]; then
```

Comment:
@angelabriel Funny I did a similar commit for angi :) Could you add an "-n" as option for echo, so we also get rid of the wrong \n in the hexdump?

Reply:
thanks for the remainder regarding '-n', completely forgot about this.
Done with current commit

Reply:
Resolved by last commit